frontend/test/metabase/scenarios/admin/people/people.cy.spec.js

```diff
@@ -322,6 +322,9 @@ describeEE("scenarios > admin > people", () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
+  });
+
+  it("should unsubscribe a user from all subscriptions and alerts", () => {
```

Comment:
Move setup related to subscription inside this test since I'm adding more EE test in this describe block.